showcase-app/plugin/identityPlugin.js

```diff
@@ -1,8 +1,27 @@
-import { QueryEngine } from '@comunica/query-sparql';
-
-const EXTENSION_ID = 'lfmjmfikelnaphplfdkbkmepacogciai';
+/**
+ * To communciate with the extension, we need to know its extension ID
+ * A Chrome Extension ID is the first 32 characters of the SHA256 hash of a public key, where characters 0-9a-f are translated to their respective a-p counterparts.
+ * The public key official public key is generated by Google as soon as the extension is uploaded to the Chrome Web Store.
+ * In the meantime, we can use a temporary public key to test the extension locally with a fixed extension id.
+ */
+const EXTENSION_ID = 'jhojffnfpoedbmolmgnddndjhmonmhnl';
 const DEBUG = true;
 
+/**
+ * @typedef {object} Identity
+ * @property {string} displayName - the display name of the identity
+ * @property {string} idpOrWebID - the IDP or WebID of the identity
+ * @property {object} metadata - metadata of the identity
+ * @property {string} metadata.name - name of the identity
```

Comment:
Where is this coming from?

Reply:
this is jsdoc type definition that is used for the `onIdentityChanged` callback function and also for the  `updateProfile`
I've now moved it close to its actual use